deprecation_notifier/DeprecationNotifier/DNAppDelegate.m

```diff
@@ -39,9 +39,20 @@ @implementation DNAppDelegate
 
 - (void)applicationDidFinishLaunching:(NSNotification *)aNotification {
   NSString *expectedVersion = NSLocalizedString(@"expectedVersion", @"");
+  NSString *expectedBuildsStr = NSLocalizedString(@"expectedBuilds", @"");
+  NSArray *expectedBuilds;
```

Comment:
nit: For large expectedBuilds builds list, a set theoretically has faster lookups.

```objc
NSString *buildsKey = @"expectedBuilds";
NSString *builds = NSLocalizedString(buildsKey, @"");
NSSet *expectedBuilds = [NSSet setWithArray:[builds componentsSeparatedByString:@","]];
if (!expectedBuilds.count || [expectedBuilds containsObject:buildsKey]) expectedBuilds = nil;
```